splash/tests/test_render.py

```diff
@@ -589,6 +586,18 @@ def _do_same_requests(self, url, params, other_format):
         return r1, r2
 
 
+class RenderVectorPngTest(RenderPngTest):
+    def request(self, query, *args, **kwargs):
+        query.setdefault('scale_method', 'vector')
+        return super(RenderVectorPngTest, self).request(query, *args, **kwargs)
+
+
+class RenderVectorJsonTest(RenderJsonTest):
```

Comment:
Why is it necessary to run render.json tests with scale_method=vector, what do they check which is not checked in render.png tests?